<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartWindow</class>
 <widget class="QMainWindow" name="StartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>331</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="FirstLevel">
       <property name="text">
        <string>Level 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SecondLevel">
       <property name="text">
        <string>Level 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ThirdLevel">
       <property name="text">
        <string>Level 3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="About">
       <property name="text">
        <string>About</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Quit">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
